eventing/sources/README.md

```diff
@@ -0,0 +1,17 @@
+# Event Sources
```

Comment:
I would switch `Sources.md` and `README.md` such that the tool should generate `README.md` (possibly with the updating information in a comment) and then a `maintenance.md` file or the like describing how to update `README.md`.

Reply:
Done.